lyrics-fetcher-neteasecloud.el

```diff
@@ -0,0 +1,216 @@
+;;; lyrics-fetcher-neteasecloud.el --- Fetch lyrics from music.163.com -*- lexical-binding: t -*-
+
+;; Copyright (C) 2021 Korytov Pavel
+;; Copyright (C) 2021 Syohei YOSHIDA
+;; Copyright (C) 2021 Eli Qian
+;; Copyright (C) 2014-2021 Free Software Foundation, Inc.
```

Comment:
Yeah, also the FSF copyright isn't necessary here.

It was related to a function in `lyrics-fetcher-genius.el` copied from the core Emacs for the backward compatibility.

Reply:
I've meant only `;; Copyright (C) 2014-2021 Free Software Foundation, Inc.` is unnecessary, you can leave all the rest 🙂